File: src/main/java-graphql/io/github/graphql/resolver/AssignableResolver.java
package io.github.graphql.resolver;

import io.github.graphql.model.*;

/**
 * Resolver for Assignable
 */
@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:48+0800"
)
public interface AssignableResolver {

    @javax.validation.constraints.NotNull
    UserConnectionTO assignees(AssignableTO assignableTO, String after, String before, Integer first, Integer last) throws Exception;

}

File: src/main/kotlin-graphql/io/github/graphql/kotlin/resolver/AssignableResolver.kt
package io.github.graphql.kotlin.resolver

import io.github.graphql.kotlin.model.*

/**
 * Resolver for Assignable
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-07T19:48:14+0800"
)
interface AssignableResolver {

    @Throws(Exception::class)
    fun assignees(assignableTO: AssignableTO, after: String?, before: String?, first: Int?, last: Int?): UserConnectionTO

}
